src/components/cart/cart.module.scss

```diff
@@ -0,0 +1,23 @@
+.cartPageContainer {
+  padding-top: 50px;
+
+  .table {
+    width: 99%;
```

Comment:
Looks wrong. Better to use width: calc(100% - 20px) for example
On some devices, it can be without margins at all